recipe/meta.yaml

```diff
@@ -48,3 +49,4 @@ extra:
     - dcherian
     - malmans2
     - aulemahal
+    - kthyng
```

Comment:
@kthyng are you OK being a maintainer for this recipe?

Reply:
Yes, thank you!

Reply:
thank you!